sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -334,6 +335,71 @@ class SQLContext(@transient val sparkContext: SparkContext)
     applySchema(rowRDD, appliedSchema)
   }
 
+  @Experimental
+  def load(
+    dataSourceName: String,
+    options: Map[String, String]): DataFrame = {
```

Comment:
No we don't but I think there is any reason to force the syntactic overhead of saying `Map(...)` on the user.  If they put double keys in a `Map` they will also be silently discarded so I don't think we are regressing at all.